gtp_cmd_handlers.py

```diff
@@ -277,7 +277,6 @@ def cmd_genmove(self, color=None):
 
         root = self._player.get_root()
         dbg("")
-        dbg(root.describe())
```

Comment:
I thought this was required for minigui

Reply:
No, Minigui only actually parses lines that are printed to stderr and start with the prefix `mg-`, everything else is just echoed to it's console. We added the `root.describe()` call just for something interesting to show in the console, it's not needed at all.